tests/testthat/test-impute.r

```diff
@@ -0,0 +1,227 @@
+context("Checking that columns and dataframes are imputed correctly")
```

Comment:
I like these tests and I appreciate the lentils, roombas, and ligers hiding in healthcareai.

Could you add a test to check that feeding imputeDF a list of new values works (e.g., if the mode of a column in develop doesn't actually show up in deploy)? It looks like all the cases where you give imputeVals to imputeDF, these were originally computed using the same dataframe.